### PR TITLE
New Article: Firewall manager v2 change log

### PR DESCRIPTION
**Pull request by DonNateR**
From Pr #1926 

- Add editorial review

- Add images

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-4e85db57e7/how-to/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-d03f07b1a4/how-to/).

**Comment by kmsholcomb** [comment]
Hi @DonNateR. Not many edits on this one! See the attached Word doc (or PDF copy of it). Let me know if you have any questions or feedback. Thanks!

[Firewall Manager v2 change log_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/866311/Firewall.Manager.v2.change.log_editsKH.pdf)
[Firewall Manager v2 change log_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/866312/Firewall.Manager.v2.change.log_editsKH.docx)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-d5444db2ae/how-to/)

File: content/dedicated-hosting/firewall-manager-v2-change-log.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+permalink: firewall-manager-v2-change-log/
+audit_date: '2017-03-24'
+title: Firewall Manager v2 change log
+type: product
```
Comment (stephamon) [inline content/dedicated-hosting/firewall-manager-v2-change-log.md]: should be article, not product. I think I've got this wrong in the templates, so I'll fix it

Comment (DonNateR) [inline content/dedicated-hosting/firewall-manager-v2-change-log.md]: I'll see if it's the same in Contributing as well.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-c4a947a5a5/how-to/)

**Comment by stephamon** [comment]
👍

**Pull request merged**